sympy/utilities/_compilation/util.py

```diff
@@ -162,6 +164,7 @@ def glob_at_depth(filename_glob, cwd=None):
     globbed = []
     for root, dirs, filenames in os.walk(cwd):
         for fn in filenames:
+            # This is not tested:
```

Comment:
Why is this?

Reply:
Also I believe this whole function can be replaced with the standard library glob once we drop Python 2, because of the recursive glob support.

Reply:
The line below uses `fnmatch` which isn't imported and wasn't even imported when the function was added. This function has clearly never been used or tested because it would never have worked.